web/react-web/src/Graph/Graph.js

```diff
@@ -0,0 +1,241 @@
+                }
+
+                name = data.description.replace('%PREDICTION_DATE%', `<br/>${new Date(data.prediction_date).toLocaleDateString()}`);
+
+                zones.push({
+                    value: predictionXValue,
+                    dashStyle: 'line'
+                });
+                dashStyle = 'dash';
+            } else {
+                name = `Active cases for ${data.long_name}`;
+            }
+
+
+            const lineData = data.date_list.map((date, index) => {
+                const y = data.values[index];
+
+                let x = null;
+                if (xAxisRelative) {
+                    x = index + 1;
+                } else {
+                    x = Date.parse(date);
+                }
+
+                let marker = null;
+                let enabled = null;
+
+                if (x === maxXValue) {
+                    enabled = true;
+                    marker = {
+                        symbol: 'triangle-down',
+                            enabled: true,
+                            radius: 8
+                    };
+                }
+
+                if (x === predictionXValue) {
+                    enabled = true;
+                    marker = {
+                        symbol: 'diamond',
+                        enabled: true,
+                        radius: 8
+                    };
+                }
+
+                return {
+                    enabled, x, y, marker
+                };
+            });
+
+            series.push({
+                type: 'line',
+                dashStyle,
+                name,
+                marker: {
+                    enabled: data.type !== 'prediction'
+                },
+                data: lineData,
+                color: colors[series.length % colors.length],
+                zones,
+                zoneAxis: 'x'
+            });
+        });
+
+        finalOptions.title = {
+            text: `Active cases and prediction for ${countries.join(', ')}`
+        };
+
+        finalOptions.series = series;
+
+        switch (axesType) {
+            default:
+            case AXES_LINEAR:
+                finalOptions.xAxis.type = 'linear';
+                finalOptions.yAxis.type = 'linear';
+                break;
+            case AXES_LOG:
+                finalOptions.yAxis.type = 'logarithmic';
+                finalOptions.xAxis.type = 'linear';
+                break;
+            case AXES_LOG_LOG:
+                finalOptions.xAxis.type = 'logarithmic';
+                finalOptions.yAxis.type = 'logarithmic';
+                break;
+        }
+
+        if (xAxisRelative) {
+            finalOptions.xAxis.title.text = 'Day since relevant number of cases';
+
+            finalOptions.tooltip.formatter = function() {
+                //https://api.highcharts.com/highcharts/tooltip.formatter
+                if (this.points) {
+                    //shared tooltip, we need extract data from every serie
```

Comment:
Nit: `serie` -> `series` (the word has the same singular and plural)

Reply:
Fixed.